src/components/Comments/WriteComment.js

```diff
@@ -68,7 +68,7 @@ const WriteComment = ({
     if (isKeyboardOpen) {
       commentInputRef.current.focus();
     }
-  }, [isKeyboardOpen, commentInputRef.current]);
+  }, [isKeyboardOpen, commentInputRef?.current]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change to the `useEffect` dependency array is a good one. It's important to note that React's `useEffect` hook will only re-run when its dependencies change. Previously, if `commentInputRef.current` was undefined during the initial render and then defined later, the effect wouldn't run again because `commentInputRef.current` was not tracked as a dependency. Now, with optional chaining (`?.`), it correctly tracks changes to `commentInputRef.current`.
```diff
-  }, [isKeyboardOpen, commentInputRef.current]);
+  }, [isKeyboardOpen, commentInputRef?.current]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->